sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -115,6 +115,11 @@
      */
     private Set<AppCenterService> mServices;
 
+    /**
+     * Started services for which the log isn't sent yet.
+     */
+    private List<String> mStartedServicesNamesToLog;
```

Comment:
You can use Collection

Reply:
`StartServiceLog` required `List` which in turn is required for `JSONUtils`. So, this change may affect some other code. `List` looks good for me here, let me know if you will nor change your mind.